autogluon/searcher/searcher.py

```diff
@@ -80,9 +80,8 @@ def get_best_reward(self):
            Assumes higher reward values indicate better performance.
         """
         with self.LOCK:
-            if len(self._results) > 0:
-                config = max(self._results, key=self._results.get)
-                return self._results[config]
+            if self._results:
+                return max(self._results.values())
         return 0.0
```

Comment:
should we also use -inf everywhere else there are zeros?  
It seems we should be consistent.

Reply:
yep, definitely, thanks for comment.